### ISSUE TITLE
Unable to reproduce HLS translation example in README

### ISSUE THREAD

**Issue by gabrielrodcanal** [hanchenye/scalehls#68]
Hello,
I'm trying to reproduce the example provided for Polybench in your README, but I'm getting the following error:
```
pre-translate.mlir:12:58: error: expected operation name in quotes
          %2 = hls.affine.select #set(%arg5) %1, %0 : f32 {timing = #hls.t<10 -> 10, 0, 0>}
```

I'm copy pasting the commands in the "Compiling HLS C/C++" section.

**Comment by HahaLan97** [comment]
> i think mlir dialect always require the variable datatype to be at the end of the line so for example:
> ```%2 = hls.affine.select #set(%arg5) %1, %0 : f32 {timing = #hls.t<10 -> 10, 0, 0>} should be```
> ```%2 = hls.affine.select #set(%arg5) %1, %0 {timing = #hls.t<10 -> 10, 0, 0>}: f32```
You can find this solution in issue [#58 ](https://github.com/hanchenye/scalehls/issues/58), but the cause for this error is still unkonwn.

**Comment by hanchenye** [comment]
Fixed in b69f334c9c122fad3759fcf388d3f039689b6fc0.
